libexec/bats

```diff
@@ -20,6 +20,8 @@ help() {
   echo "  -h, --help     Display this help message"
   echo "  -p, --pretty   Show results in pretty format (default for terminals)"
   echo "  -t, --tap      Show results in TAP format"
+  echo "  -x, --extended Show results in extended TAP format"
+  echo "  -u, --junit    Show results in JUnit format, hudson compatible"
```

Comment:
I'd remove what's after the comma entirely. Whether it's for Hudson, Jenkins or whatever, it's a JUnit format, and it's all that matters :)